install/index.markdown

```diff
@@ -7,13 +7,13 @@ slug: install
 title: Install
 ---
 
-> MoveIt! is currently developed against **Ubuntu and ROS Indigo and Jade**. **ROS Kinetic** is not yet released. It is recommended that you move to ROS Jade for the latest features.
+# Binary Installation Instructions
 
-**Developers:**: see [source install](source_install.html) instructions.
+MoveIt! is released every month or so into Ubuntu Debians via the ROS infrastructure. For more information see the [target platforms](http://www.ros.org/reps/rep-0003.html).
```

Comment:
rephrased to "Ubuntu debian packages", similar to http://wiki.ros.org/kinetic/Installation/Ubuntu